airflow/executors/base_executor.py

```diff
@@ -310,3 +313,14 @@ def debug_dump(self):
             len(self.event_buffer),
             "\n\t".join(map(repr, self.event_buffer.items())),
         )
+
+    def send_callback_to_execute(self, request: CallbackRequest) -> None:
+        """Sends callback for execution.
+
+        Provides a default implementation which sends the callback to DagProcessor.
+
+        :param request: Callback request to be executed.
+        """
+        if not self.callbacks_sink:
+            raise ValueError("Callback sink is not ready.")
+        self.callbacks_sink.send_callback_to_execute(request)
```

Comment:
I think we don't need to keep the `send_callback_to_execute` here 
```suggestion
        self.callbacks_sink.send(request)
```

(Or if you want slightly more verbose `send_callback`)

Reply:
Done